### PR TITLE
Add sign-in using firebase authentication

### PR DESCRIPTION
**Pull request by indhu-gunda**
capture (3)

### CODE REVIEW

File: frontend/src/redux/rootReducer.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import {combineReducers} from 'redux';
 import museumArtworks from './museumArtworkReducer';
 import userArtworks from './userArtworkReducer';
+import {firebaseReducer} from 'react-redux-firebase';
 
 export default combineReducers({
   museumArtworks,
   userArtworks,
+  firebase: firebaseReducer,
```
Comment (indhu-gunda) [inline frontend/src/redux/rootReducer.js:9]: I'm confused because there is a trailing comma, and when I run eslint locally it's fine.

Comment (isabelsang) [reply frontend/src/redux/rootReducer.js:9]: That's so weird.  It also is displaying errors on UserUploadForm.js but I saw that you pushed a commit for eslint fixes so maybe git is outdated for some reason?
image

File: frontend/src/components/dashboard/account-components/AvatarHeader.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const auth = useSelector((state) => state.firebase.auth);
+  return (
+    <>
+      <Divider />
+      <div className={classes.root}>
+        <Avatar
+          className={classes.avatar}
+          src={auth.photoURL}
+        />
+        <div className={classes.div} />
```
Comment (isabelsang) [inline frontend/src/components/dashboard/account-components/AvatarHeader.js:34]: What is this div for?

Comment (indhu-gunda) [reply frontend/src/components/dashboard/account-components/AvatarHeader.js:34]: it's for in between and it didn't add space properly when I added to other divs.

Comment (indhu-gunda) [reply frontend/src/components/dashboard/account-components/AvatarHeader.js:34]: but I can rename the class so its purpose is more obvious

**Pull request merged**